.gitignore

```diff
@@ -13,3 +13,4 @@ diff.png
 /test/symgroups.pdf
 /test/screenshotter/unicode-fonts
 coverage/
+*~
```

Comment:
This is for emacs backup files.

Reply:
Should probably add this to your global gitignore